ganttproject-tester/test/net/sourceforge/ganttproject/resource/TestHumanResourceManager.java

```diff
@@ -0,0 +1,62 @@
+package net.sourceforge.ganttproject.resource;
```

Comment:
Please add a copyright notice (don't forget to replace <YOUR_NAME_HERE> in the snippet below with your name):

```
/*
Copyright 2021 BarD Software s.r.o, <YOUR_NAME_HERE>

This file is part of GanttProject, an open-source project management tool.

GanttProject is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
 the Free Software Foundation, either version 3 of the License, or
 (at your option) any later version.

GanttProject is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with GanttProject.  If not, see <http://www.gnu.org/licenses/>.
*/

```

Reply:
Done!